portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,14 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const facts =
+      ['I am Romanian.', 'Guitar and piano are some instruments that I play.', 'I am really tall (180 cm)', 'Friends call me by my middle name.'];
```

Comment:
I generally recommend that when creating lists in code, put each list item on its own line.

This avoids long lines (compare https://google.github.io/styleguide/jsguide.html#formatting-column-limit).

It also makes it easier to merge when list items are added or removed in different pull requests (this is less of a concern here, I understand).